config/initializers/auth0.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider(
+    :auth0,
+    ENV["AUTH0_CLIENT_ID"],
+    ENV["AUTH0_CLIENT_SECRET"],
+    ENV["AUTH0_DOMAIN"],
```

Comment:
Do you think these hash values are critical as well?
```suggestion
    ENV.fetch("AUTH0_CLIENT_ID"),
    ENV.fetch("AUTH0_CLIENT_SECRET"),
    ENV.fetch("AUTH0_DOMAIN"),
```